controllers/configmap_controller.go

```diff
@@ -85,7 +86,15 @@ func NewConfigMapReconciler(mgr manager.Manager, clusterConfig cluster.Config, w
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to initialize Prometheus configuration")
 	}
-	svcData, err := servicescm.NewData(&[]servicescm.Service{}, &[]servicescm.FileInfo{})
+	svcData, err := servicescm.NewData(&[]servicescm.Service{{
+		Name:                         windows.WindowsExporterServiceName,
+		Command:                      windows.WindowsExporterServiceCommand,
+		NodeVariablesInCommand:       nil,
+		PowershellVariablesInCommand: nil,
+		Dependencies:                 nil,
+		Bootstrap:                    false,
+		Priority:                     1,
```

Comment:
Could you explain how you settled on priority of 1? Just thinking ahead, will all bootstrap services share priority of 0?

Reply:
With the pattern you are introducing, `NewConfigMapReconciler()` will grow very large as we keep adding service. I suggest encapsulating this with  a helper. Ideally we should be using a constant that this helper method uses to generate the data.